assets/js/vue/FileManager.vue

```diff
@@ -83,10 +114,21 @@ let fileManagerResolve = [];
 let totalFiles = 0;
```

Comment:
Can remove itemsChanged variable, can move totalFiles and doneFiles inside initDropzone()